README.md

```diff
@@ -82,25 +100,41 @@ from [Frontegg Portal Domain](https://portal.frontegg.com/development/settings/d
 ```groovy
     dependencies {
     // Add the Frontegg Android Kotlin SDK
-    implementation 'com.frontegg.sdk:android:1.+'
+    implementation 'com.frontegg.sdk:android:LATEST_VERSION'
```

Comment:
Switched to `LATEST_VERSION` because of the Android Studio message 👇🏽  that appears whenever you use `+` anywhere
<img width="910" alt="Screenshot 2024-10-07 at 12 25 04 PM" src="https://github.com/user-attachments/assets/d38b4c07-9039-4e5c-849f-8b41a2bfc588">